cypress/integration/GlossaryAppIntegration/GlossaryAppIntegration.js

```diff
@@ -0,0 +1,152 @@
+/// <reference types="Cypress" />
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Given('user is on home page', () => {
```

Comment:
Should this really be here and not in common?

Reply:
gone completely